lib/Model/Exception/CouldNotInstallExtension.php

```diff
@@ -2,8 +2,13 @@
 
 namespace Phpactor\Extension\ExtensionManager\Model\Exception;
 
+use Throwable;
 use RuntimeException;
 
 class CouldNotInstallExtension extends RuntimeException
 {
+    public function __construct(string $message, Throwable $previous = null)
```

Comment:
I was hesitant here to "break" compatibility and ask only the previous exception, as the first argument given here is basically `$previous->getMessage()`... we could use `(?Throwable $previous = null) { parent::__construct($previous !== null ? $previous->getMessage() : 'Could not install', 0, $previous); }` or something along those lines, wdyt ?

Reply:
I guess we could also throw this exception when there was no previous exception, so this is OK I think.

I'd also be happy to not override the consructor and just pass `new CouldNotInstallException($message, 0, $previous)`